classes/class06/hw6/routes/index.js

```diff
@@ -0,0 +1,104 @@
+var routes = {};
+
+var mongoose = require('mongoose');
+var Twot = require('../models/twotModel.js')
+var User = require('../models/userModel.js')
+
+routes.home = function(req, res){
+	// 	Twot.remove({}, function(err) { 
+ //   		console.log('collection removed') 
+	// });
+	// User.remove({}, function(err) { 
+ //   		console.log('collection removed') 
+	// });
+	Twot.find({}).sort({"_id": -1}).exec(function(err, alltwot) {
+		User.find({}).exec(function(err, allusers) {
+			var qu = User.where({logged: true});
+			qu.findOne(function (err, mU) {
+				if (err) return handleError(err);
```

Comment:
Funny, I couldn't find where this was defined. I believe you made it though and this is a good idea.